client/src/components/AuditAdmin/Setup/Review/Review.test.tsx

```diff
@@ -1024,4 +1025,28 @@ describe('Audit Setup > Review & Launch', () => {
       expect(warning).toHaveClass('bp3-intent-warning')
     })
   })
+
+  it('shows a spinner while sample sizes are computed', async () => {
+    jest.useFakeTimers()
+    const expectedCalls = [
+      apiCalls.getSettings(settingsMock.full),
+      apiCalls.getJurisdictions({
+        jurisdictions: jurisdictionMocks.allManifests,
+      }),
+      apiCalls.getJurisdictionFile,
+      apiCalls.getContests(contestMocks.filledTargetedWithJurisdictionId),
+      aaApiCalls.getSampleSizes(sampleSizeMock.calculating),
+      aaApiCalls.getSampleSizes(sampleSizeMock.ballotPolling),
+    ]
+    await withMockFetch(expectedCalls, async () => {
+      renderView()
+      await screen.findByRole('heading', { name: 'Sample Size' })
+      screen.getByText('Loading sample size options...')
+      jest.advanceTimersByTime(1000)
+      await screen.findByText(
+        'Choose the initial sample size for each contest you would like to use for Round 1 of the audit from the options below.'
+      )
+    })
+    jest.useRealTimers()
```

Comment:
I don't feel strongly about this but we could use a `beforeEach` to reset the timers before every test

Reply:
That's what I would normally do, but I figured since it's just one test in this file I'd try out putting it specifically in that test